src/controllers/playback/video/index.js

```diff
@@ -1410,6 +1412,20 @@ export default function (view) {
             chapterThumbText = document.createElement('h2');
             chapterThumbText.classList.add('chapterThumbText');
             chapterThumbTextContainer.appendChild(chapterThumbText);
+            chapterThumbName = document.createElement('span');
+            chapterThumbName.classList.add('chapterThumbName');
+            chapterThumbTextContainer.appendChild(chapterThumbName);
```

Comment:
I think this will be redesigned at some point. We just should keep them consistent.
As an option, we could use ellipsis.
Also, we could add a footer below the thumbnail.